src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -43,6 +44,7 @@ function getProps(
     accountName: 'Chase (...2345)',
     institutionName: 'Chase',
     fiatAccountType: FiatAccountType.BankAccount,
+    fiatAccountSchema: FiatAccountSchema.AccountNumber,
```

Comment:
Just to be clear, all the updates related to adding `fiatAccountSchema` are from the version bump on `fiatconnect-types`, correct? And aren't actually related to the SIWE auth work? (Same goes for the `TransferType` field on quotes.)

Reply:
yes, it was a side effect from bumping FC SDK, which uses FC types v7